e2e-cypress/files/support/e2e.ts

```diff
@@ -8,8 +8,8 @@ beforeEach(function() {
 })
 
 afterEach(function() {
-  const testName = this.currentTest.fullTitle().replace(/ /g, '_');
-  const fileName = `system-output-${testName}.txt`;
+  const testName = this.currentTest?.fullTitle().replace(/ /g, '_');
```

Comment:
I'd rather default that to an explicit string instead of having the `String(testName)` one lines below.
```suggestion
  const testName = this.currentTest?.fullTitle().replace(/ /g, '_') && 'unknown';
```
Mind the `&&` instead of the `??`. This should handle cases where the test name would be an empty string.

Reply:
Just one correction: We have to use `||` to do replace falsy values.